src/phoenix/core/evals.py

```diff
@@ -171,3 +173,58 @@ def get_document_evaluation_scores(
                 if result.HasField("score") and document_position < num_documents:
                     scores[document_position] = result.score.value
         return scores
+
+    def export_evaluations(self) -> List[Evaluations]:
+        evaluations: List[Evaluations] = []
+        evaluations.extend(self._export_span_evaluations())
+        evaluations.extend(self._export_document_evaluations())
+        return evaluations
+
+    def _export_span_evaluations(self) -> List[SpanEvaluations]:
+        span_evaluations = []
+        for eval_name, span_evaluations_by_id in self._span_evaluations_by_name.items():
```

Comment:
nit. let's make a copy before each iteration
```suggestion
        with self._lock:
            span_evaluations = tuple(self._span_evaluations_by_name.items())
        for eval_name, span_evaluations_by_id in span_evaluations:
```

Reply:
addressed in [8be7d5e](https://github.com/Arize-ai/phoenix/pull/2094/commits/8be7d5e9c55527ffa3a1c2e9b6ebf3345209dc1e)